src/styles/bloks/calendar.scss

```diff
@@ -0,0 +1,39 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $calendar-width;
+  gap: 1px;
+  padding: 70px 10px;
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: $container-day-width;
+    height: $container-day-height;
+    border: 1px solid black;
+  
+    &:first-child {
+      margin-left: $margin-fist-child;
+    }
+  
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translate(0, -20px);
+    }
+  
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+      }
+    }
+
+    @for $index from 28 through 31 {
+      &--month-duration--#{$index} &__cell:nth-child(n + #{$index + 1}) {
+        display: none;
+      }
+    }
```

Comment:
Please, fix this loop, as it doesn't work right (cellss are not disepearing whem I change a modificator). I think that part with "&__cell:nth-child(n + #{$index + 1})" should be sfter {
Just pay attention to it